python/dlisio/lis/load.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from .. import core
+from .file import logical_file, physical_reel
+
+def load(path):
+    """ Loads a file and returns one filehead
+
+    Parameters
+    ----------
+
+    path : str_like
+        path to lis-file
+
+
+    Returns
+    -------
+
+    lis : dlisio.lis.physicalfile
+
+    """
+    f = core.open(path, 0)
+    tm = core.read_tapemark(f)
+    f.close()
+
+    offset = 0
+    tapeimage = core.valid_tapemark(tm)
+    if tapeimage and tm.type == 1: offset += 12
```

Comment:
2) Because some files start's with a TM with type 1, and then lfp reports EOF straight away. We should make some "find_first_type_0_tm" or something to that effect.